File: pi.c
#include <stdio.h>

double pi()
{
	double sum = 0;
	int i;

	for (i = 0; i < 1e8; i++) {
		sum += 4.0 / (i * 2 + 1) * (i % 2 ? -1 : 1);
	}

	return sum;
}

int main()
{
	printf("%.15f\n", pi());
	return 0;
}

File: pi.js

function pi()
{
	var sum = 0, i = 0;
	
	while (i < 1e9) {
		sum += 4 / (i * 2 + 1) * (i % 2 ? -1 : 1);
		i++;
	}

	return sum;
}

console.log(pi());

